inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -55,6 +57,10 @@ endif()
 set (PYTHON_BRIDGE_SRC_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 add_subdirectory (src/openvino/inference_engine)
 
+if(ENABLE_WHEEL)
+    add_subdirectory(install/wheel)
```

Comment:
Since this duplicates install logic for OpenVINO components (provides alternative distribution set)  it should be a **separate** CMake script. Please do not make it as part of a regular build by adding conditional _add_subdirectory_.

Reply:
it is disabled by default + we can exclude this target from ALL.

Reply:
@echuraev, @ilya-lavrenov  what is your opinion here, should we exclude this target from all and build it separatly?

Reply:
how much time does it take to build this target? E.g. all binaries are built and we build this target on top of it.

Reply:
<30 seconds

Reply:
In this case we can build with all targets..